packages/database/src/migrations/20201126234247-ConvertDhisSurveysIntoTupaia-modifies-data.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+import { arrayToDbString, generateId, insertObject, updateValues } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const surveyCodes = ['RHC', 'RHFSC', 'RHFSCTS'];
+
+const switchServiceType = async (db, code, serviceType) => {
+  await updateValues(db, 'data_source', { service_type: serviceType }, { code });
+};
+
+const getDataSources = async (db, codes) => {
+  const surveyCodeQuery = codes.map(e => `'${e}'`).join(',');
```

Comment:
No need to change your code here, but there's also `arrayToDbString` in the migration utilities which does the same thing - we use that a lot!